### PR TITLE
feat: added home page

### PR DESCRIPTION
**Pull request by nakulsharma15**
Created Landing/Home page of Altur Store

index-1
index-2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,128 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://altur-ui.netlify.app/stylesheets/main.css">
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Comment (kjais1720) [inline index.html:13]: import this link in the `main.css` file, which will be imported in all the pages, this way you won't need to import the same link in every page

File: index.html [hunk 2]
```diff
@@ -1 +1,128 @@
...
+                    <button class="cart-btn btn outline-secondary-btn icon-btn"><i class="fas fa-shopping-cart"></i> Add to Cart</button>
+                </div>
+            </div>
+
+        </div>
+
+    </div>
+
+    <div class="footer flex-center">
+        <p class="text-l bold"> Made by <a href="https://www.linkedin.com/in/nakulsharma15"> Nakul Sharma</a></p>
```
Comment (shubhamkumar648) [inline index.html]: add trget attribute

File: index.html [hunk 3]
```diff
@@ -1 +1,128 @@
...
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <title>Altur Store</title>
+</head>
+
+<body>
+
+    <nav class="flex-center-se">
```
Comment (kjais1720) [inline index.html]: It is not very clear with the class name as to what it does, is it `justify-content: center or space-evenly` or `align-items: center or space-evenly`. You can create classes like, `justify-content-sp-ev | sp-bw | center | start | end`

**Review by shubhamkumar648 — APPROVED** [review]
good design, code looks clean to me, consider my comment changes , you can merge after changes

File: index.html (hunk 2 above)
Comment (nakulsharma15) [reply index.html]: I'll add it. Thanks for pointing out.

**Review by kjais1720 — APPROVED** [review]
Code looks clean and readable with good use of utility classes, 👍

File: index.html (hunk 1 above)
Comment (nakulsharma15) [reply index.html:13]: Ok, I'll do it. Thanks for pointing out

File: index.html (hunk 3 above)
Comment (nakulsharma15) [reply index.html]: It's for space-evenly. I'll change the class name.

**Pull request merged**